src/ComponentConnectorRect.js

```diff
@@ -52,16 +52,41 @@ export class MatrixCell extends React.Component {
   }
 
   render() {
+    const inverted = this.props.item[1] > 0.5;
+
+    let color = "#838383";
+
+    if (inverted) {
+      color = "#DE4B39";
+    }
+
+    if (this.props.item[0] > 1) {
```

Comment:
I think the logic here needs revising to make this a darker color, rather than overriding inversion red.